qt/aqt/main.py

```diff
@@ -848,7 +848,7 @@ def maybe_check_for_addon_updates(self) -> None:
 
         if elap > 86_400:
             check_and_prompt_for_updates(
-                self, self.addonManager, self.on_updates_installed
+                self, self.addonManager, self.on_updates_installed, routine=True
             )
```

Comment:
I presume 'routine' is intended to distinguish automatic checks from the user clicking the 'check for updates' button? If so, perhaps a name like 'requested_by_user' might be clearer?